src/content/guides/build-performance.md

```diff
@@ -152,6 +155,61 @@ new CommonsChunkPlugin({
 })
 ```
 
+### Avoid Extra Optimization Steps
+
+webpack does extra algorithmic work to optimize the output for size and load performance. These optimizations are performant for smaller codebases, but can be costly in larger ones:
+
+```js
+{
+  optimization: {
+    removeAvailableModules: false,
+    removeEmptyChunks: false,
+    splitChunks: false,
+  }
+}
+```
+
+### Output Without Path Info
+
+webpack has the ability to generate path info in the output bundle. However, this puts garbage collection pressure on projects that bundle thousands of modules. Turn this off in the `options.output.pathinfo` setting:
+
+```js
+{
+  output: {
+    pathinfo: false
+  }
+}
+```
+
+### Node.js Version
+
+There has been a [performance regression](https://github.com/nodejs/node/issues/19769) in the latest stable versions of Node.js with regard to the ES2015 Map and Set implementation. A fix has been merged in master, but a release has yet to be made. In the meantime, to get the most out of incremental build speeds, try to stick with version 8.9.x (problem exists between 8.9.10 - 9.11.1). webpack has moved to using those ES2015 data structures liberally, and it will improve the initial build times as well.
+
+### TypeScript
```

Comment:
We hit a markdownlint limitation (https://github.com/markdownlint/markdownlint/issues/175), it doesn't like headers with the same content, this title (TypeScript) and the one from Specific Tooling Issues section.

I am not sure how we could rename these headers to comply.
Maybe changing "TypeScript" to "With TypeScript" in the last section?
Or add "via ts-loader" to this heading: TypeScript via ts-loader?

Reply:
I called it "Typescript Loader"